frontend/rolecall/src/app/user/user-editor.component.scss

```diff
@@ -1,5 +1,6 @@
 @import "src/colors";
 @import "src/constants";
+@import "src/std";
```

Comment:
Is this actually used anywhere?

Reply:
Yes, this is where I put my standard style classes.

Reply:
What I meant was I don't see anything from that file being used in `user-editor` styles, so it looks like it's imported unnecessarily?

Reply:
I was going include stc/std in most scss files so that they are hooked up properly as we put more code into std.scss.